### PR TITLE
Back notifications

### PR DESCRIPTION
**Pull request by NikolaHadzicNs**

### CODE REVIEW

**Comment by qsnnikola** [comment]
- Code is clean and seems ok to me.
- My question: What happens when back button is pressed and collecting of data is running? 
(Notification shows and then quickly disappears?)

**Review by qsnnikola — APPROVED** [review]
Code is clean and seems ok to me.
My question: What happens when back button is pressed and collecting of data is running?
(Notification shows and then quickly disappears?)

**Review by markokatic0001 — APPROVED** [review]

**Comment by NikolaHadzicNs** [comment]
Thank you for your reviews. I had problem when back is pressed because it calls onPause. I solved that problem in a meantime. I will commit and push today.

File: app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java [hunk 1]
```diff
@@ -908,4 +926,32 @@ public void openGPSSettings() {
         Intent i = new Intent(ACTION_LOCATION_SOURCE_SETTINGS);
         startActivityForResult(i, Constants.REQUEST_GPS_SETTINGS);
     }
+
+    private void backgroundTrackNotification(int notificationID){
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
+        builder.setSmallIcon(R.drawable.ic_launher);
+        builder.setContentTitle("GetGoing notification");
+        builder.setContentText("Location tracking in background");
```
Comment (SlobodanPrijic) [inline app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java]: Text should not be hardcoded. Please define string in xml that can be localized.

File: app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java [hunk 2]
```diff
@@ -908,4 +926,32 @@ public void openGPSSettings() {
...
+        builder.setAutoCancel(true);
+        builder.setDefaults(Notification.DEFAULT_VIBRATE);
```
Comment (SlobodanPrijic) [inline app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java]: I think silent notification would be better, no need for vibrating phone.

File: app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java [hunk 3]
```diff
@@ -908,4 +926,32 @@ public void openGPSSettings() {
...
+
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0, this.getIntent(), PendingIntent.FLAG_UPDATE_CURRENT);
+
+        builder.setContentIntent(pendingIntent);
+
+
+        notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.notify(notificationID,builder.build());
+        notificationExist = true;
+
```
Comment (SlobodanPrijic) [inline app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java]: @NikolaHadzicNs remove those empty lines

Comment (NikolaHadzicNs) [reply app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java]: I removed empty lines and vibration and defined text in xml.

**Review by SlobodanPrijic — APPROVED** [review]

**Pull request merged**